src/pages/index.astro

```diff
@@ -119,5 +158,13 @@ import { Image } from "astro:assets";
             <p>John Tan-Aristy</p>
          </div>
       </div>
+      
+      <hr />
+      
+      <!-- TODO: This really should be in the footer. -->
+      <div id="privacy" class="mx-12 mt-12 text-neutral-700 text-sm">
+        By submitting your email, you consent to us collecting your email, IP address and approximate location based on your IP address to prevent spam and send you one email when Zenith sign ups open.
```

Comment:
Todo - make a new page for privacy and change the link on the footer to this. On that privacy page, we'll have the cabin link && ^^